internal/stack/_static/geoip_source/GeoLite2-City-Test.json

```diff
@@ -0,0 +1,13063 @@
+[
+   {
+      "192.0.2.0/24" : {
+         "city" : {
+            "geoname_id" : 5506956,
+            "names" : {
+               "en" : "Las Vegas"
```

Comment:
I've tried to use other "real" cities here, and get the data about using 
https://www.geonames.org/

This data is just faked for testing purposes in `elastic-package`.